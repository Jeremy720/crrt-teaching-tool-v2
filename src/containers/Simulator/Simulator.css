.Simulator {
  height: 100%;
}

.CRRT-title {
  background-color: #c8e9ff;
  color: #3b8ed6;
  height: 38px;
  margin: 0;
  padding: 21px;
}

.dataOutputContainer {
  display: flex;
  height: 100vh;
  width: 100vw;
}

.btn-active, 
.data-output-btn, 
.physical-exam-output-btn {
  height: 100%;
  outline: none;
  /* width: 160px; */
}

.btn-active {
  border-bottom: solid 2px #000;
  border-left: solid 2px #000;
  border-right: solid 2px #000;
  border-top: solid 2px #000;
  font-weight: bold;
}

.data-output-btn {
  background-color: #fff;
  border-left: solid 1px #000;
  border-right: solid 1px #000;
  color: #000;
  font-size: 1rem;
}

.physical-exam-output-btn {
  background-color: #fff;
  border-bottom: solid 1px #000;
  border-left: solid 1px #000;
  border-right: solid 1px #000;
  border-top: solid 1px #000;
  color: #000;
}

.modalBtnContainer {
  background-color: #EFF7FE;
  display: flex;
  flex-direction: column;
  height: 80%;
  justify-content: center;
  width: 250px;
}

.modal-container {
  height: 80.5%;
  width: 82.35%;
}

.simulator-header {
  background-color: #c8e9ff;
  border-bottom: solid 1px #3b8ed6;
}

.form-buttons-container {
  background-color: #c8e9ff;
  width: 55%;
  margin: 0 auto;
  display: flex;
  align-items: center;
  justify-content: space-around;
}



.orders-btn {
  background-color: #f00;
  border-radius: 5rem;
  color: #fff;
  cursor: pointer;
  font-size: 1rem;
  height: 2rem;
  margin: 1rem 1rem 1rem 0;
  outline: none;
  width: 5rem;
}

.orders-btn:hover {
  box-shadow: 0px 0px 10px #d0d6d3;
  background-color: #3b8ed6;
}

.crrt-display-btn, .restart-case-btn {
  color: #4c4646;
}

.crrt-display-btn:hover, .restart-case-btn:hover {
  color: #fff;
  background-color: #4c4646;
}

/*Can comment this out if it just doesn't end up working*/
.simulator {
  /* align-items: center;
  display: flex;
  flex-direction: column;*/
  height: 100%;
  /*justify-content: center;
  position: absolute;*/
  width: 100%;
  /*z-index: 1;*/
}

body, #root, .App {
  width: 100vw;
}