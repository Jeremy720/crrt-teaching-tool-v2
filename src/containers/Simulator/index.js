import React from 'react';
import './Simulator.css';

function Simulator() {
  return(
    <div>
      <h1>CRRT SIMULATOR!</h1>
    </div>
  )
}

export default Simulator;