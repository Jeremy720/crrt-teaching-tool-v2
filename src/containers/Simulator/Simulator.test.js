import React from 'react';
import Simulator from './';
import { shallow } from 'enzyme';

describe('Simulator', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Simulator />)
  });

  it('matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  });

});