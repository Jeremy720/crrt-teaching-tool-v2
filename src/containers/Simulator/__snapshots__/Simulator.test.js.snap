// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Simulator matches the snapshot 1`] = `ShallowWrapper {}`;
